RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GoogleMapsHelper.swift
+//  RoutesApp_ios
+//
+//  Created by user on 19/9/22.
+//
+
+import Foundation
+import GoogleMaps
+
+class GoogleMapsHelper {
+    static var shared = GoogleMapsHelper()
+
+    func drawPolyline(map: GMSMapView, list: [Coordinate], hexColor: String = "#004696") {
```

Comment:
add this default color as a constant

Reply:
Added the default polyline color as constant